cudamapper/src/cudamapper_utils.hpp

```diff
@@ -49,5 +49,29 @@ namespace cudamapper
 /// \param unfused_overlaps vector of overlaps, sorted by (query_id, target_id) combination and query_start_position
 void fuse_overlaps(std::vector<Overlap>& fused_overlaps, const std::vector<Overlap>& unfused_overlaps);
 
+/// \brief Given a string s, produce its kmers (length <kmer-length>) and return them as a vector of strings.
+/// \param s A string sequence to kmerize.
+/// \param kmer_length A kmer length to use for producing kmers.
+/// \param stride The number of bases to skip when selecting kmers (most often, this should be equal to 1).
+std::vector<std::string> kmerize_string(const std::string& s, int kmer_length, int stride);
+
+/// \brief Given two sorted vectors of comparable types, return a size_t count of the number of shared elements.
```

Comment:
@vellamike It says `Given two sorted vectors`, but yes, I would also specify it in `\param` (see my comment in function definition for an idea how to avoid this constraint)

Reply:
Fixed in 3b805e8e.